docs/tutorial/tutorial-0.rst

```diff
@@ -67,12 +73,8 @@ Next, install the additional dependencies needed for your operating system:
 
     Building BeeWare apps on macOS require:
 
-    * **Git**, a version control system. You can download Git from
-      `git-scm.org <https://git-scm.com/download/>`__.
-
-    * **Xcode**, Apple's IDE tooling. Xcode is available for free
-      from `the macOS App Store
-      <https://apps.apple.com/au/app/xcode/id497799835?mt=12>`__.
```

Comment:
A full Xcode installation is not required unless the user wants to build iOS apps, so at this stage it's better to just use the command line developer tools, because they're much smaller and much less of a barrier to entry.